### ISSUE TITLE
Bug Report: Word Eaters Chaos Lord unable to take Juggernaut of Khorne

### ISSUE THREAD

**Issue by MisoPhat** [BSData/wh40k-7th-edition#3590]
File/Catalogue: Chaos Space Marines: Codex (2012) v2019 / CSM World Eaters Butcherhorde

BattleScribe version:** 2.00.15

Platform: Windows/iOS

Dropbox: Yes

Description: A Chaos Lord with Mark of Khorne in the World Eaters Butcherhorde Supplement is unable to select a Juggernaut of Khorne as a mount.

**Comment by cartag** [comment]
Duplicate of #3588, already resolved just waiting on release.

**Comment by MisoPhat** [comment]
Thank you for the fast response and awesome job you all do. Sorry for the repost, will check next time
